include/seqan3/alphabet/aminoacid/all.hpp

```diff
@@ -0,0 +1,111 @@
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
+
+/*!\file alphabet/aminoacid/all.hpp
+ * \ingroup aminoacid
+ * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
+ * \brief Meta-header for the aminoacid submodule; includes all headers from alphabet/aminoacid/.
+ */
+
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
+
+/*!\defgroup aminoacid
+ * \brief Contains the amino acid alphabet, the reduced amino acid alphabet \TODO and translation from nucleotide
+ * into amino acid
+ * \ingroup alphabet
+ *
+ * \par Introduction
+ * Amino acid sequences are an important part of bioinformatic data processing and used by many applications
+ * and while it is possible to represent them in a regular std::string, it makes sense to have specialised data
+ * structures in most cases. This sub-module offers the 27 letter aminoacid alphabet as well as a reduced version
+ * that can be used with regular container and ranges.
+ *
+ * | Letter   | Description     |
+ * |:--------:|-----------------|
+ * |   A      | Alanine         |
+ * |   C      | Cysteine        |
+ * |   D      | Aspartic acid   |
+ * |   E      | Glutamic acid   |
+ * |   F      | Phenylalanine   |
+ * |   G      | Glycine         |
+ * |   H      | Histidine       |
+ * |   I      | Isoleucine      |
+ * |   K      | Lysine          |
+ * |   L      | Leucine         |
+ * |   M      | Methionine      |
+ * |   N      | Asparagine      |
+ * |   O      | Pyrrolysine     |
+ * |   P      | Proline         |
+ * |   Q      | Glutamine       |
+ * |   R      | Arginine        |
+ * |   S      | Serine          |
+ * |   T      | Threonine       |
+ * |   U      | Selenocysteine  |
+ * |   V      | Valine          |
+ * |   W      | Tryptophan      |
+ * |   Y      | Tyrosine        |
+ * |   B      | N or D          |
+ * |   J      | I or L          |
+ * |   Z      | Q or E          |
+ * |   *      | Terminator      |
+ * |   X      | any aminoacid   |
+ *
+ * While the proteinogenic amino acid alphabet consists of 22 amino acids, there are more characters defined
+ * which offer a certain ambiguity.
+ *
+ * \par Printing and conversion to char
+ *
+ *
+ * | to_char()        | seqan3::aa27 |
+ * |:----------------:|:------------:|
+ * | type::A          |    `'A'`     |
+ * |   ...            |    ...       |
+ * | type::Z          |    `'Z'`     |
+ * | type::TERMINATOR |    `'*'`     |
+ * | type::X          |    `'X'`     |
+ * | type::UNKNOWN    |    `'X'`     |
+ *
+ * \par Assignment
+ *
+ * | assign_char()    | seqan3::aa27      |
+ * |:----------------:|:-----------------:|
+ * |   `'A'`          |  aa27::A          |
+ * |   ...            |    ...            |
+ * |   `'Z'`          |  aa27::T          |
+ * |   `'*'`          |  aa27::TERMINATOR |
+ * |   `'X'`          |  aa27::X          |
+ * | e.g. `'!'`       |  aa27::X          |
```

Comment:
I think the above two tables are not too relevant here, because there is nothing that needs to be explained. 
[for nucleotides these paragraphs underline the differences between the types]